.github/workflows/build-and-test.yaml

```diff
@@ -46,6 +46,8 @@ jobs:
               exit 1
           fi
 
+          sleep 8
```

Comment:
While this appears to solve the immediate problem, adding a call to `sleep` in a test is always going to come back to haunt you.

What about something like this instead:
```bash
curl --retry 4 --retry-connrefused localhost:9117/metrics -s > /dev/null
```

Reply:
Thanks, that's a nice suggestion.